csaf_2.1/prose/edit/src/guidance-on-size.md

```diff
@@ -36,7 +36,7 @@ All _CSAF producers_ SHOULD NOT produce CSAF documents which exceed those limits
 ## File size
 
 A CSAF document in the specified JSON format encoded in UTF-8 SHOULD conform to known size limits of current technologies parsing JSON content,
-e.g.: 15 MB.
+e.g.: 50 MiB.
 
 > At least one database technology in wide use for storing CSAF documents rejects insert attempts when
```

Comment:
I will amend the following note, as the relation between the 50 Megabytes and the 16 Megabytes is not as clear, as the former reasoning of less than 16 Megabytes leading to a 15 Megabytes soft limit.